chromium_src/chrome/browser/ui/toolbar/app_menu_model.cc

```diff
@@ -13,6 +13,9 @@
 #define IDS_NEW_INCOGNITO_WINDOW_OLD IDS_NEW_INCOGNITO_WINDOW
 #undef IDS_NEW_INCOGNITO_WINDOW
 #define IDS_NEW_INCOGNITO_WINDOW IDS_BRAVE_NEW_INCOGNITO_WINDOW
+#define IDS_NEW_INCOGNITO_TAB_OLD IDS_NEW_INCOGNITO_TAB
```

Comment:
Can we just define this IDS_BRAVE_NEW_PRIVATE_TAB ?

Reply:
Now I've removed the `IDS_NEW_INCOGNITO_TAB_OLD` definition and just undef the `IDS_NEW_INCOGNITO_TAB` to avoid getting re-declaration error. Kindly check.